mj_envs/robot/robot.py

```diff
@@ -437,7 +437,7 @@ def get_visual_sensors(self, height:int, width:int, cameras:list, device_id:int,
                 current_sensor_value[cam_name] = data
 
                 # calibrate sensors
-                for cam in device['cam']:
+                for cam in device['cams']:
```

Comment:
This should be cam. 
Example see here - https://github.com/vikashplus/mj_envs/blob/v0.4dev/mj_envs/envs/fm/assets/franka_robotiq.config#L41

Reply:
When we call `['cam']`, it threw a key not found error which was fixed by `['cams']`

Reply:
Can you send me the exact command so that I can reproduce it at my end.